oem/ibm/requester/dbus_to_file_handler.cpp

```diff
@@ -142,6 +143,9 @@ void DbusToFileHandler::processNewResourceDump(
             resDumpProgressIntf);
     }
 
+    info("Received acf file path {ACF_FILE_PATH} and vsp string {VSP_STRING}",
+         "ACF_FILE_PATH", acfFilePath, "VSP_STRING", vspString);
```

Comment:
Most of the cases, this ACF file path will be empty unless its a targetted acf flow. Do we really need to trace this  or remove it ?